module/applications/sheets/item-actor-details-sheet.mjs

```diff
@@ -121,7 +121,10 @@ export default class CrucibleActorDetailsItemSheet extends CrucibleBaseItemSheet
   async #onDropTalent(event) {
     const data = CONFIG.ux.TextEditor.getDragEventData(event);
     const talents = this.document.system.talents;
-    if ( (data.type !== "Item") || talents.has(data.uuid) ) return;
+    const hasLeveledTalents = this.document.type === "archetype";
```

Comment:
For the very short term, can we leave a `// TODO [issue number here]` anywhere that we have to logically condition on whether we have leveled talents or not so we can identify all the places that we need to circle back to later?